niworkflows/viz/utils.py

```diff
@@ -199,20 +199,45 @@ def extract_svg(display_object, dpi=300, compress='auto'):
 def cuts_from_bbox(mask_nii, cuts=3):
     """Finds equi-spaced cuts for presenting images"""
     from nibabel.affines import apply_affine
-    mask_data = mask_nii.get_data()
-    B = np.argwhere(mask_data > 0)
-    start_coords = B.min(0)
-    stop_coords = B.max(0) + 1
```

Comment:
Or maybe this should have just been `stop_coords = B.max(0)`?

Reply:
This indeed seemingly fixed the issue (that was the first thing I tried). However, being at it, I wanted to improve the visualization because such a liberal calculation usually gave almost empty views of the axial plane in many images.